README.md

```diff
@@ -90,6 +91,47 @@ summon python listEC2.py
 `python listEC2.py` is the command that summon wraps. Once the Python program exits,
 the secrets stored in temp files and in the Python process environment are gone.
 
+### `secrets.yml` Flags
+
+Currently, you can define how the value of a variable will be processed using YAML tags. Multiple
+tags can be defined per variable by spearating them with `:`. By default, values are resolved
+as literal values.
+- `!file`: Resolves the variable value, places it into a tempfile, and returns the path to that
+file.
+- `!var`: Resolves the value as a variable ID from the provider.
+- `!str`: Resolves the value as a literal (default).
```

Comment:
Besides the `str` literal type, the literal types `bool`, `float`, and `int`could also be included here?

Reply:
@diverdane No, they don't need to be mentioned - they're only relevant in the internal representation and yaml will set those values automatically in most cases.

Reply:
Thanks for the explanation!